test/app/languageforge/lexicon/newLexProject/e2e/lexicon-new-project.spec.js

```diff
@@ -363,7 +363,6 @@ describe('E2E testing: New Lex Project wizard app', function () {
       expect(page.backButton.isDisplayed()).toBe(false);
       expect(page.initialDataPage.browseButton.isDisplayed()).toBe(true);
       expect(page.progressIndicatorStep3Label.getText()).toEqual('Verify');
-      page.expectFormIsNotValid();
```

Comment:
That makes sense.

It appears though that `lexicon-new-project.js` is [pretty explicit](https://github.com/sillsdev/web-languageforge/blob/e5b0aafa0f97eb7630839ad92dbe67838f4a5afc/src/angular-app/languageforge/lexicon/new-project/lexicon-new-project.js#L328) about `newProject.initialData` and `newProject.verifyData` both being always valid. I would think that would be the place to add a check and return `neutral()` in the default state. Not sure why that wasn't apparent to me earlier (Unless of course you've tried it and know that doesn't work for some reason, in which case I trust it's more complicated than it appears and won't attempt it again).